### ISSUE TITLE
Page with income/outcome

### ISSUE THREAD

**Issue by Eldenroot** [Sama34/newpoints#3]
Add a page to see income/outcome for logged user

**Comment by Sama34** [comment]
I think that the logs module from the ACP might be moved to the front end.

So users can browse through their own logs and admins would be able to browse all logs.

**Comment by Eldenroot** [comment]
Yes

**Comment by Eldenroot** [comment]
You can use something similar to MyBB reputation system 
![image](https://github.com/user-attachments/assets/83016b85-aa27-449c-84f4-30b50c5e3a2d)

**Comment by Eldenroot** [comment]
This should be added into next release, because now you have no info how you going :)

**Comment by Sama34** [comment]
This should be now fixed with the new logs page.
![image](https://github.com/user-attachments/assets/507b45ad-6f23-48fb-baff-1cc67f520371)

https://github.com/OUGC-Network/newpoints/commit/784b8dc1a68e05cb7db7b399259ce2645b104bf3

**Comment by Eldenroot** [comment]
Is possible to add a summary above the log table? Total income, total outcome?

**Comment by Eldenroot** [comment]
Would be great to combine with your newest bank plugin :)

**Comment by Sama34** [comment]
> Is possible to add a summary above the log table? Total income, total outcome?

Yes, but maybe a new issue for that because I might take some before I work on it.

> Would be great to combine with your newest bank plugin :)

Yes, the Bank plugin integrates into this new logging system.

Regards.

**Comment by Eldenroot** [comment]
Hi, please which template to use for displaying custom text and total points in newpoins.php? I mean to display my content (bar) below the navigation bar for all /newpointa.php

**Comment by Sama34** [comment]
Please elaborate, drop me a DM at discord if you want.

I think this issue can be closed now that the logging system was updated to implement your original suggestiion.

**Comment by Eldenroot** [comment]
Yes, this issue is somehow solved.

Anyway I need to display balance here:

![Image](https://github.com/user-attachments/assets/d3d47cb3-c255-4f84-998c-a4231af14dc1) above the content abd below the navigation
